### DISCUSSION TITLE
HIP SDK 5.5.0 and tons of errors in Visual Studio 2022

### DISCUSSION THREAD

**Discussion by Unknowpers** [ROCm/ROCm#2492] [Q&A]
Hello everyone
Im new to HIP but have experiences with CUDA. Just installed HIP SDK 5.5.0 windows and created a project from sdk's templates (empty) in visual studio 2022. the moment i make a cpp file and add hip_runtime.h header, like 1040 errors of intellisense appear about some headers in some headers not being found, some common words (like __global__) not being known and etc. its only for cpp files and on hip files, theres no error and therefore no intellisense.
ofcourse its not interfering with compilation and it goes without any error but intellisense is not working correctly. its a huge pain to not have this thing working.

anyone any idea how it can be fixed ?

Heres a list of these errors:
[error_list.txt](https://github.com/RadeonOpenCompute/ROCm/files/12682246/error_list.txt)

**Comment by Azbeltk** [comment]
I don't know if it's because of these issues but the 5.5 windows installer is no longer available. Just today I was setting up a windows system with a 6900xt but couldn't find the HIP SDK installer cause the link just takes you to amd.com instead of fetching the file.

**Reply by Unknowpers** [reply]
well, i got it fixed by suppressing intellisenses errors. also took me a while to understand that i need to add architecture code name in a certain field in projects settings of visual studio to get it working on radeon448sp and ryzen5-5500u -_-
looks like cant expect much from it "on windows" for now even though it worked nicely at the end.
